tensorflow/lite/micro/tools/ci_build/test_esp32.sh

```diff
@@ -26,34 +26,34 @@ pwd
 source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
 
 TARGET=esp
+TARGET_ARCH=xtensa-esp32
```

Comment:
Without specifying this the resultant generated example directory has xthe wrong name.  It doesn't detect the arch right and has x86 in it instead of what we want.